plans/session-recording.fmf

```diff
@@ -1,9 +1,18 @@
-summary:
-  Tests using pre-recorded responses from external services.
-prepare:
-  how: ansible
-  playbooks:
-  - files/packit-testing-farm-prepare-session-recording.yaml
+summary: Session recording testcases
+discover:
+  how: fmf
+  filter: tier:2
 execute:
-  script:
-  - make check TESTS_TARGET=tests_recording
+  how: tmt
+adjust:
+  when: "distro <= rhel-8 or distro <= centos-8"
```

Comment:
Just note: No need to have quotes here.

Reply:
yep, but for YAML I'm sometimes not sure about used chars, so in this case I prefer to use quotes although it is not so necessary.